src/airflow/.env

```diff
@@ -0,0 +1,8 @@
+AIRFLOW_UID=1126896676
+AIRFLOW_IMAGE_NAME=extending_airflow:latest
+GOOGLE_LOCAL_CREDENTIALS_PATH=~/.config/gcloud
+GOOGLE_DOCKER_CREDENTIALS_PATH=/.config/gcloud
+GOOGLE_APPLICATION_CREDENTIALS=/.config/gcloud/service_account_credentials.json
+AIRFLOW_CONN_GOOGLE_CLOUD_DEFAULT='google-cloud-platform://?extra__google_cloud_platform__key_path=/.config/gcloud/service_account_credentials.json'
+GCP_PROJECT_ID=open-targets-genetics-dev
+#GCP_GCS_BUCKET
```

Comment:
It was a reminder that it can be set up in the same way so it's available to every DAG. I'll remove it and we can keep this in mind for further parametrisation.